cves/CVE-2012-0031.yml

```diff
@@ -48,20 +50,12 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 1db31e0b0aa70725929e72f7289a113cc5d9b511
-     note:
-   - commit: dcf8bc4eb10bd25d772babcccd87a92cb110ec7b
-     note:
-   - commit: b0835c731a7d7f65b5d409f42b886fc8e3111bfc
+   - commit: 65718cf0f5980f1e5e2e9a2335a0cfb599b30cda
      note:
```

Comment:
Was there no commit message with the fix? I'm curious what was changed to fix an invalid memory free call. Was the fix to simply not allowing the users to access the field that causes the error or was the fix more directly involved in changing how the function accessed memory?